UPGRADE-5.0.md

```diff
@@ -42,6 +42,25 @@ Validator
  * Calling `EmailValidator::__construct()` method with a boolean parameter has been removed, use `EmailValidator("strict")` instead.
  * Removed the `checkDNS` and `dnsMessage` options from the `Url` constraint.
 
+Form
+----
+
+ * Using callable strings as `empty_data` in `ChoiceType` has been deprecated in Symfony 4.1 use a `\Closure` instead.
+
+  Before:
+  
+  ```php
+  'empty_data' => 'SomeValueObject::getDefaultValue',
+  ```
+
+  After:
+  
+  ```php
+  'empty_data' => function (FormInterface $form, $data) {
+      return SomeValueObject::getDefaultValue();
+  },
```

Comment:
I think, you should rather use a global function call as an example. A static method call can still be written as `['SomeValueObject', 'getDefaultValue']` and I personally would prefer that notation over an anonymous function.

Reply:
And in vain you prefer such a call to static functions.
This leads to the fact that the connection to the real function is lost and this is guaranteed to lead to problems during refactoring.

Reply:
Well, it was not my intention to discuss my personal preferences. Shouldn't have mentioned it. Still, the static call adds unnecessary distraction to the example, imho.